src/utils/formatDate.js

```diff
@@ -0,0 +1,13 @@
+export function formatDate(date) {
+  const dateFormatOptions = {
+    year: "2-digit",
+    month: "2-digit",
+    day: "2-digit",
+  };
+
+  const formattedDate = new Intl.DateTimeFormat("de", dateFormatOptions).format(
```

Comment:
@ray-wtag vai, `de` is actually german format. We are not using `en-US` because it shows the date in `mm.dd.yy` format whereas in the design it is in `dd.mm.yy` format. 

Sujon vai actually recommended to use the builtin date formatter instead of third party libraries.

Reply:
Okaty got it, Thanks.